tests/testthat/_snaps/verb-arrange.md

```diff
@@ -55,7 +52,6 @@
       <SQL>
       SELECT *
       FROM `df`
-      ORDER BY `a`
```

Comment:
Not sure whether `arrange()` should actually "unarrange" or not. At least there was a PR for this #471

Reply:
Agreed that this behaviour seems more consistent. Allow it maybe seems like it should still generate a warning?